src/main/java/com/fundation/webservice/model/CriteriaAudio.java

```diff
@@ -67,27 +67,27 @@ public int getAudioBitRate() {
     }
 
 	/**
-     * Method that modifies the it rate for output file.
+     * Method that modifies the bitrate for output file.
      *
-     * @param time The audioBitRate parameter define the number of bit rate for output file.
+     * @param audioBitRate The audioBitRate parameter define the number of bitrate for output file.
      */
     public void setAudioBitRate(int audioBitRate) {
         this.audioBitRate = audioBitRate;
     }
 
 	/**
```

Comment:
Align the comments.

Reply:
There's nothing to align. This is the standard javadoc layout.